packages/server/src/api/routes/record.js

```diff
@@ -36,7 +36,7 @@ router
   )
   .post(
     "/api/:modelId/records/validate",
-    authorized(WRITE_MODEL, ctx => ctx.params.modelId),
```

Comment:
Should this function to get the `modelId` out of the ctx have been removed here?

Reply:
Not sure, don't remember touching this at all